benchadapt/python/tests/test_run.py

```diff
@@ -87,8 +87,8 @@ def test_name_defaulting(self, monkeypatch):
         monkeypatch.delenv("CONBENCH_PROJECT_COMMIT", raising=False)
 
         run = BenchmarkRun(reason=run_json["reason"])
+        assert run.github == {}
```

Comment:
This is an internal, not-so-important state.

Do we need to do more testing on the return value of `to_publishable_dict()` in the future?

Reply:
Yes, I'd rather test the output of `to_publishable_dict()` than the attributes on the dataclass.